NEWS.md

```diff
@@ -1,5 +1,9 @@
 # chromote (development version)
 
+* `ChromoteSession` now records the `targetId`. This eliminates one round-trip 
+  to the browser when viewing or closing a session, and will make it possible to
+  re-start a closed session (#94).
```

Comment:
For consistency with the rest of this file, can you change this to not be hard-wrapped?